render/jpql/src/test/kotlin/com/linecorp/kotlinjdsl/render/jpql/serializer/impl/JpqlDerivedEntitySerializerTest.kt

```diff
@@ -0,0 +1,142 @@
+package com.linecorp.kotlinjdsl.render.jpql.serializer.impl
+
+import com.linecorp.kotlinjdsl.querymodel.jpql.entity.Entities
+import com.linecorp.kotlinjdsl.querymodel.jpql.entity.impl.JpqlDerivedEntity
+import com.linecorp.kotlinjdsl.querymodel.jpql.select.impl.JpqlSelectQuery
+import com.linecorp.kotlinjdsl.render.TestRenderContext
+import com.linecorp.kotlinjdsl.render.jpql.serializer.JpqlRenderClause
+import com.linecorp.kotlinjdsl.render.jpql.serializer.JpqlRenderSerializer
+import com.linecorp.kotlinjdsl.render.jpql.serializer.JpqlRenderStatement
+import com.linecorp.kotlinjdsl.render.jpql.serializer.JpqlSerializerTest
+import com.linecorp.kotlinjdsl.render.jpql.writer.JpqlWriter
+import io.mockk.*
+import io.mockk.impl.annotations.MockK
+import org.assertj.core.api.WithAssertions
+import org.junit.jupiter.api.Test
+
+@JpqlSerializerTest
+class JpqlDerivedEntitySerializerTest : WithAssertions {
+    private val sut = JpqlDerivedEntitySerializer()
+
+    @MockK
+    private lateinit var writer: JpqlWriter
+
+    @MockK
+    private lateinit var serializer: JpqlRenderSerializer
+
+    @Test
+    fun handledType() {
+        // when
+        val actual = sut.handledType()
+
+        // then
+        assertThat(actual).isEqualTo(JpqlDerivedEntity::class)
+    }
+
+    @Test
+    fun `serialize - WHEN statement is select and clause is not select, THEN draw only alias`() {
```

Comment:
How about referencing [the test code](https://github.com/line/kotlin-jdsl/blob/develop/3.0.0/render/jpql/src/test/kotlin/com/linecorp/kotlinjdsl/render/jpql/serializer/impl/JpqlAliasedExpressionSerializerTest.kt#L36-L41) for the parameterized test?

You can see from the test name that the `statement` is a `select`, but the `clause` is not a `select`. However, the test does not test for all clauses.

Therefore, I would like to suggest a Parameterized Test with the following code.
```kotlin
@ParameterizedTest
@StatementClauseSource(
    includes = [StatementClause(statement = JpqlRenderStatement.Select::class)],
    excludes = [StatementClause(clause = JpqlRenderClause.Select::class)],
)
```